pkgs/development/python-modules/sphinxext-opengraph/default.nix

```
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "sphinxext-opengraph";
-  version = "0.8.2";
+  version = "0.9.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/wpilibsuite/sphinxext-opengraph/blob/fd959a790ea8146dfbffb68534cccea83c86a7e1/setup.py#L41